### ISSUE TITLE
Lost Connection!

### ISSUE THREAD

**Issue by alduxvm** [Gremsy/gSDK#4]
Hi!

Using gremsy T3v3, testing firmwares:
* v750
* v754
* v756
* v757
* v758

I'm connecting the gimbal to port COM2 and I'm using a usb to serial cable for my computer.

I enabled the MAVLINK control using the gTuneDesktop v1.4.5 app, my gimbal show the next in windows:

![Screen Shot 2021-05-20 at 8 46 51 AM](https://user-images.githubusercontent.com/8823379/118981137-08b81e00-b948-11eb-845c-1c845f72c4c5.png)

When trying the SDK in linux, I always have the problem of `Lost Connection!`, the port in linux is `/dev/ttyUSB0`:

```
~/gSDK/build$ ./gSDK -d /dev/ttyUSB0
OPEN PORT
Connected to /dev/ttyUSB0 with 115200 baud, 8 data bits, no parity, 1 stop bit (8N1)

START READ THREAD 

 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
 Lost Connection!
```

I tried all above firmwares and not luck...

What to do?

**Comment by clolsonus** [comment]
One thing you could double check is that your TX / RX lines are correct.  I had an issue with a drone-code standard uart cable (6-pin) between my pixhawk4 and the gimbal until I realized the TX/RX lines were reversed on the gimbal side.  I've come to expect peripherals to just plug in and work, so this one caught me off guard.

**Comment by rrr6399** [comment]
Do you have to use the UART interface of the Gremsy with a FTDI cable? Can the gSDK work using the Gremsy's usb connection?
(I'm thinking not, since I'm getting the "Lost Connection" message when I attempt to connect to /dev/USB0. There is a serial entry under /dev/serial/id so it seems that Ubuntu is recognizing it as a serial connection.)

**Comment by gyillikci** [comment]
@rrr6399 yes, you need to connect gnd, tx, rx to a ttl to usb in order to get data. gSDK doesnt use the usb connection(it is the pitfall I also fell in)
so the solution is as below for t3v3:

![image](https://user-images.githubusercontent.com/5228976/157046647-8d922125-0825-4577-8842-a356f8449f73.png)
